src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -228,6 +285,39 @@ export class NetCoreDebugHelper implements DebugHelper {
 
         return pathNormalize(result, platformOS);
     }
+
+    private async InstallDebuggerOnContainer(containerName: string): Promise<string> {
+        const debuggerPath: string = '/remote_debugger';
+        const installUnZip: string = '/bin/sh -c "apt-get update && apt-get install unzip"';
+        const installDebugger: string = `/bin/sh -c "curl -sSL https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l ${debuggerPath}"`;

```

Comment:
Does this cover all of the basic .NET Core image flavors (e.g. slim, alpine, bionic)?  For example, I recall having to install `curl` on some base images.

Reply:
I haven't tested all flavors. I added a TODO item to add validation on target OS and flavor and stop the debugging.

Reply:
The new script will handle ubuntu, debian and alpine flavors.
But no validation yet for windows or other flavors and will be handled later.